libraries/Microsoft.Bot.Builder/Streaming/StreamingRequestHandler.cs

```diff
@@ -381,11 +396,9 @@ private async Task ReconnectAsync(IDictionary<string, string> requestHeaders = n
                 }
             }
 
-            // The ServiceUrl of a streaming connection follows the pattern "urn:[ChannelName]:[Protocol]:[Host]".
-            var uri = ServiceUrl.Split(':');
-            var protocol = uri[uri.Length - 2];
-            var host = uri[uri.Length - 1];
-            await clientWebSocket.ConnectAsync(new Uri(protocol + host + _reconnectPath), CancellationToken.None).ConfigureAwait(false);
+            // TODO: Set the authentication header if it wasn't passed in.
```

Comment:
I don't love TODOs is this a 4.6 TODO?

Reply:
We need to figure out the story for reconnect in general. I think the headers should always be passed in, so we don't end up with an ABS specific implementation, but we need to decide on the appropriate place to set them and a default behavior. For 4.6 there will be no channels with support for the reconnect endpoint.